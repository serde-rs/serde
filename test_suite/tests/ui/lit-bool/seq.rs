use serde_derive::Deserialize;

#[derive(Debug, Deserialize)]
#[serde(tag = "subtype")]
enum Child {
    Topic, Sidebar
}

#[derive(Debug, Deserialize)]
#[serde(tag = "type", seq_form="xx")]
enum ParentNoSeq {
    Title,
    #[serde(untagged)]
    SubStructure(Child),
}

#[derive(Debug, Deserialize)]
#[serde(tag = "type", seq_form="true")]
enum E2 {
    Title,
    #[serde(untagged)]
    SubStructure(Child),
}

#[derive(Debug, Deserialize)]
#[serde(tag = "type", seq_form=true)]
enum Ok1 {
    Title,
    #[serde(untagged)]
    SubStructure(Child),
}

#[derive(Debug, Deserialize)]
#[serde(tag = "type", seq_form=false)]
enum Ok2 {
    Title,
    #[serde(untagged)]
    SubStructure(Child),
}

fn main() {}
