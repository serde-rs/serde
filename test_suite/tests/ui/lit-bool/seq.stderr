error: expected serde content attribute to be a bool: `content = false/true`
  --> tests/ui/lit-bool/seq.rs:10:32
   |
10 | #[serde(tag = "type", seq_form="xx")]
   |                                ^^^^

error: expected serde content attribute to be a bool: `content = false/true`
  --> tests/ui/lit-bool/seq.rs:18:32
   |
18 | #[serde(tag = "type", seq_form="true")]
   |                                ^^^^^^
